examples/ex-01-refinement.c

```diff
@@ -107,6 +107,7 @@ my_Step(braid_App        app,
    /* Use backward Euler to propagate solution */
    /* Use forward Euler to estimate the local trucation error */
    (u->value) = 1./(1. + dt)*v;
```

Comment:
I would argue that this option is a bit more interesting than the zero solution for the periodic case, but I don't have a strong opinion

Reply:
I'll add this for the periodic case then.